File: backend/src/main/java/nl/wilcokas/luckystackworker/dto/PSFDTO.java
package nl.wilcokas.luckystackworker.dto;

import lombok.AllArgsConstructor;
import lombok.Builder;
import lombok.Data;
import lombok.NoArgsConstructor;
import nl.wilcokas.luckystackworker.model.PSFType;

@Data
@NoArgsConstructor
@AllArgsConstructor
@Builder
public class PSFDTO {
    private double airyDiskRadius;
    private double seeingIndex;
    private double diffractionIntensity;
    private PSFType type;
}

File: backend/src/main/kotlin/nl/wilcokas/luckystackworker/dto/PSFDTO.kt
package nl.wilcokas.luckystackworker.dto

import nl.wilcokas.luckystackworker.model.PSFType

data class PSFDTO(
    var airyDiskRadius: Double = 0.0,
    var seeingIndex: Double = 0.0,
    var diffractionIntensity: Double = 0.0,
    var type: PSFType? = null
)